xchange-stream-binance/README.md

```diff
@@ -0,0 +1,52 @@
+
+## Binance
+
+### Live Subscription/Unsubscription
+The feature to support Live Subscribe/Unsubcribe to streams has been added in version 5.0.7.
```

Comment:
would probably eliminate this version `5.0.7` in case version released differs

Reply:
Done